tests/e2e/TestConstants.ts

```diff
@@ -17,12 +17,26 @@ function getBaseUrl(): string {
     return baseUrl.replace(/\/$/, '');
 }
 
+function getKeycloackUrl(): string {
+    const keycloakUrl: string | undefined = process.env.TS_SELENIUM_KEYCLOAK_URL;
+    if (!keycloakUrl) {
+        return 'http://keycloak-url';
+    }
+
+    return keycloakUrl.replace(/\/$/, '');
```

Comment:
What is this for? I don't understand the meaning of this regex

Reply:
The same as for the "TS_SELENIUM_BASE_URL". Removal of the "/" character at the end of the introduced URL if that url has been set with that character.